[2019-04-30 17:06:56,126] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: Mixed_Failing_Demo.change_dir1 2019-04-30T17:06:38.791865+00:00 [queued]>
[2019-04-30 17:06:56,129] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: Mixed_Failing_Demo.change_dir1 2019-04-30T17:06:38.791865+00:00 [queued]>
[2019-04-30 17:06:56,129] {__init__.py:1353} INFO - 
--------------------------------------------------------------------------------
[2019-04-30 17:06:56,129] {__init__.py:1354} INFO - Starting attempt 1 of 1
[2019-04-30 17:06:56,129] {__init__.py:1355} INFO - 
--------------------------------------------------------------------------------
[2019-04-30 17:06:56,141] {__init__.py:1374} INFO - Executing <Task(BashOperator): change_dir1> on 2019-04-30T17:06:38.791865+00:00
[2019-04-30 17:06:56,141] {base_task_runner.py:119} INFO - Running: ['airflow', 'run', 'Mixed_Failing_Demo', 'change_dir1', '2019-04-30T17:06:38.791865+00:00', '--job_id', '228', '--raw', '-sd', 'DAGS_FOLDER/mixed_failing.py', '--cfg_path', '/tmp/tmpqtlgmmn7']
[2019-04-30 17:06:56,404] {base_task_runner.py:101} INFO - Job 228: Subtask change_dir1 /home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/configuration.py:214: FutureWarning: The task_runner setting in [core] has the old default value of 'BashTaskRunner'. This value has been changed to 'StandardTaskRunner' in the running config, but please update your config before Apache Airflow 2.0.
[2019-04-30 17:06:56,404] {base_task_runner.py:101} INFO - Job 228: Subtask change_dir1   FutureWarning,
[2019-04-30 17:06:56,554] {base_task_runner.py:101} INFO - Job 228: Subtask change_dir1 [2019-04-30 17:06:56,554] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-04-30 17:06:56,707] {base_task_runner.py:101} INFO - Job 228: Subtask change_dir1 [2019-04-30 17:06:56,707] {__init__.py:305} INFO - Filling up the DagBag from /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/dags/mixed_failing.py
[2019-04-30 17:06:56,729] {base_task_runner.py:101} INFO - Job 228: Subtask change_dir1 [2019-04-30 17:06:56,728] {cli.py:517} INFO - Running <TaskInstance: Mixed_Failing_Demo.change_dir1 2019-04-30T17:06:38.791865+00:00 [running]> on host localhost.localdomain
[2019-04-30 17:06:56,736] {bash_operator.py:81} INFO - Tmp dir root location: 
 /tmp
[2019-04-30 17:06:56,736] {bash_operator.py:90} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_ID=Mixed_Failing_Demo
AIRFLOW_CTX_TASK_ID=change_dir1
AIRFLOW_CTX_EXECUTION_DATE=2019-04-30T17:06:38.791865+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-30T17:06:38.791865+00:00
[2019-04-30 17:06:56,736] {bash_operator.py:104} INFO - Temporary script location: /tmp/airflowtmpptztgu4f/change_dir1hlwrx4vu
[2019-04-30 17:06:56,737] {bash_operator.py:114} INFO - Running command: cd $AIRFLOW_HOME
[2019-04-30 17:06:56,740] {bash_operator.py:123} INFO - Output:
[2019-04-30 17:06:56,741] {bash_operator.py:131} INFO - Command exited with return code 0
[2019-04-30 17:07:01,136] {logging_mixin.py:95} INFO - [2019-04-30 17:07:01,136] {jobs.py:2562} INFO - Task exited with return code 0
